client/api/src/main/java/org/opennms/velocloud/client/api/model/Path.java

```diff
@@ -0,0 +1,156 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2023 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2023 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.velocloud.client.api.model;
+
+import com.google.common.base.MoreObjects;
+
+public class Path {
+    public final String peerType;
+    public final String peerName;
+    public final String description;
+    public final String edgeLogicalId;
+    public final String deviceLogicalId;
+    public final int pathCountStable;
+    public final int pathCountUnstable;
+    public final int pathCountStandBy;
+    public final int pathCountDead;
+    public final int pathCountUnknown;
+    public final int pathCountTotal;
+
+    private Path(Builder builder) {
+        peerType = builder.peerType;
+        peerName = builder.peerName;
+        description = builder.description;
+        edgeLogicalId = builder.edgeLogicalId;
+        deviceLogicalId = builder.deviceLogicalId;
+        pathCountStable = builder.pathCountStable;
+        pathCountUnstable = builder.pathCountUnstable;
+        pathCountStandBy = builder.pathCountStandBy;
+        pathCountDead = builder.pathCountDead;
+        pathCountUnknown = builder.pathCountUnknown;
+        pathCountTotal = builder.pathCountTotal;
```

Comment:
Any non-null guaranties we can give here?

Reply:
done.